server/@types/bapv.d.ts

```diff
@@ -106,10 +106,12 @@ export type VisitSessionData = {
   visitSlot?: VisitSlot
   originalVisitSlot?: VisitSlot
   visitRestriction?: 'OPEN' | 'CLOSED'
+  visitorIds?: number[]
   visitors?: VisitorListItem[]
   visitorSupport?: VisitorSupport
   mainContact?: {
-    contact?: VisitorListItem
+    contactId?: number
+    relationshipDescription?: string
```

Comment:
Instead of storing a `VisitorListItem`, just store the contact ID and relationship type which is all that is needed.